include/earcut.hpp

```diff
@@ -124,15 +124,15 @@ void Earcut<N>::operator()(const Polygon& points) {
         do {
             x = p->x;
             y = p->y;
-            if (x < minX) minX = x;
-            if (y < minY) minY = y;
-            if (x > maxX) maxX = x;
-            if (y > maxY) maxY = y;
+            minX = (std::min)(minX, x);
```

Comment:
It is a good idea; the bad idea is to use min and max preprocessor macros in the first place.